### DISCUSSION TITLE
Discussion Topics for Aug 23, 2021 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#1926] [Show and tell]
Discussion Topics for Engineering Sync

**Comment by RickHawesUSDS** [comment]
Would like to discuss my proposal for a new internal format. 

https://github.com/CDCgov/prime-reportstream/blob/rick/proposal-11/prime-router/docs/proposals/0011-internal-formats/0011-internal-formats.md

**Reply by jimduff-usds** [reply]
@RickHawesUSDS , if there's time, could you also discuss https://github.com/CDCgov/prime-reportstream/blob/7841aa1decdf125e9c2c777bcb3be368c3c7502c/prime-router/docs/proposals/0010-translations-and-schemas/0010-translations-and-schemas.md
?  Thanks!
